source/includes/_unified_display_report.md

```diff
@@ -0,0 +1,572 @@
+## Unified Display Campaign Metrics - NEW
+
+### Resource Overview
+
+| Method | URI Format |
+|---|---|
+| GET | /client_reports/unified_display/[gmaid]?[query_params] |
+### API Name: Unified Display Report
+
+### Summary
+This is a new Display Metrics API that can be used by customers that have Display campaigns from either the ReachLocal or Gannett brand. All metrics are reported at the creative level, and aggregated at the Interval/Campaign level for ReachLocal brand or Line Item level for the Gannett brand. When data is unavailable, corresponding property will not be present in the response. For example, if we do not have data for a `creative_type` or `creative_size`, it will not be present in the response.
+
+**Note:** We recommend users to migrate to this API in place of the Display Activity API and Display Creative API as they will no longer be updated.
```

Comment:
Should the documentation on these APIs reflect the same note?  I think so.

Reply:
Similar notes have been added to the other Display APIs that have slick links to this API.